bin/build

```diff
@@ -65,15 +68,34 @@ findFile() {
   head -n 1
 }
 
-# Recursively search input directory for a package.json
+# Recursively search output directory for a package.json
 PACKAGE_PATH=$(findFile "package.json" $OUTPUT_DIR/bundle/programs/server)
+export PACKAGE_PATH
 
-if [[ $PACKAGE_PATH ]]; then
-  export PACKAGE_PATH
+NODE_VERSION=$(get-version --engine node $PACKAGE_PATH 2>/dev/null) || true
```

Comment:
I thought it would be better to use the values placed in the `package.json` even though they should be the same.